codegen/src/main/kotlin/org/web3j/openapi/codegen/GenerateOpenApi.kt

```diff
@@ -20,24 +23,32 @@ import org.web3j.openapi.codegen.utils.TemplateUtils.generateFromTemplate
 import org.web3j.openapi.codegen.utils.TemplateUtils.mustacheTemplate
 import org.web3j.openapi.codegen.web3jCodegenStuff.SolidityFunctionWrapperGenerator
 import java.io.File
-import java.nio.file.Path
+import java.nio.file.Paths
 
 class GenerateOpenApi(
     private val configuration: GeneratorConfiguration
 ) {
-    fun generateAll() {
-        generateGradleResources()
-        generateCore()
-        generateServer()
-        generateWrappers()
+
+    private val SWAGGERUI_GENERATION_TASK = "completeSwaggerUiGeneration"
+
+    fun generateAll(isCodeOnly: Boolean = false, withWrappers: Boolean = true) {
+        if (!isCodeOnly) generateGradleResources()
+        generateCore(isCodeOnly)
+        generateServer(isCodeOnly)
+        if (withWrappers) generateWrappers()
+    }
+
+    fun generateAllWithSwaggerUI() {
+        generateAll()
+        generateSwaggerUI()
     }
 
-    fun generateServer() {
-        ServerGenerator(configuration).generate()
+    fun generateServer(isCodeOnly: Boolean = false) {
+        ServerGenerator(configuration).generate(isCodeOnly)
```

Comment:
`isCodeOnly` should come in through the configuration rather than changing the method signature